rails_exercises/todo_app/.rubocop.yml

```diff
@@ -0,0 +1,12 @@
+Style/Documentation:
+  Enabled: false
+Style/ExpandPathArguments:
+  Enabled: false
+Metrics/LineLength:
+  Max: 120
+Style/SymbolArray:
+  EnforcedStyle: brackets
+Style/FrozenStringLiteralComment:
+  Enabled: false
+Style/HashSyntax:
+  Enabled: false
```

Comment:
Why did you disable this one? I think this rule is good for consistency.
Did you do that because of the generated code?
If yes, prefer to use `Exclude` directive and specify targeted exceptions.

Reply:
It seemed to me Rails are not still there(https://freelancing-gods.com/2017/07/27/friendly-frozen-string-literals.html). I see the benefits of enforcing that rule but I was too lazy to apply it to all Ruby files. Is there a way to configure it for Rails 3?

Reply:
I was talking about HashSyntax rule